<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editRecord</class>
 <widget class="QWidget" name="editRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="text">
       <string>Идентификатор</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="text">
       <string>Имя</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="text">
       <string>Фамилия</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
